src/test/java/org/openmrs/contrib/qaframework/helper/TestData.java

```diff
@@ -0,0 +1,695 @@
+package org.openmrs.contrib.qaframework.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.ws.rs.NotFoundException;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.*;
```

Comment:
Rather import the specific packages required than explicitly calling all the `util.*` packages